browser/static/js/main_zstack.js

```diff
@@ -1144,8 +1156,8 @@ function handle_scroll(evt) {
     let mod_contrast = -Math.sign(evt.originalEvent.deltaY) * 4;
     // stop if fully desaturated
     current_contrast = Math.max(current_contrast + mod_contrast, -100);
-    // stop at 5x contrast
-    current_contrast = Math.min(current_contrast + mod_contrast, 400);
+    // stop at 8x contrast
+    current_contrast = Math.min(current_contrast + mod_contrast, 700);
```

Comment:
Instead, we can define global variables `const MAX_CONTRAST = 700` and `MAX_CONTRAST = -100` to make this easier to configure.